cluster/examples/kubernetes/ceph/cluster.yaml

```diff
@@ -68,8 +68,18 @@ spec:
     # deployed) to set rulesNamespace for all the clusters. Otherwise, you will get duplicate alerts with multiple alert definitions.
     rulesNamespace: rook-ceph
   network:
-    # toggle to use hostNetwork
-    hostNetwork: false
+    # supported provider are 'host' and 'multus'
+    #provider: multus
```

Comment:
Host networking is a common enough example, how about a separate example? For example:
```
# enable host networking
#provider: host
# EXPERIMENTAL: enable the Multus network provider
#provider: multus
#selectors:
...
```